go.mod

```diff
@@ -4,34 +4,34 @@ go 1.23.0
 
 require (
 	github.com/cespare/xxhash v1.1.0
-	github.com/goccy/go-yaml v1.15.15
+	github.com/goccy/go-yaml v1.17.1
 	github.com/gorilla/websocket v1.5.3
-	github.com/mattn/go-sqlite3 v1.14.24
-	github.com/quic-go/quic-go v0.43.0
+	github.com/mattn/go-sqlite3 v1.14.28
+	github.com/quic-go/quic-go v0.43.0 // No upgrade
```

Comment:
why no upgrade? some incompatibility?

Reply:
I remember Junxiao mentioned that after 43.0 there is some Cgo issue that makes web-transport unable to compile.
Also, the webtransport-go package is unfunded for now, which makes it possibly incompatible with latest version: https://github.com/quic-go/webtransport-go/issues/164

We need more investigation on this.

Reply:
It seems working after I patched the main branch of webtransport-go.